include/ofxPixels.h

```diff
@@ -78,7 +78,7 @@ typedef struct OfxRGBColourF {
 
 /** @brief Defines a double precision floating point component RGB pixel */
 typedef struct OfxRGBColourD {
-  double r, g, b, a;
+  double r, g, b;
```

Comment:
oh yes this is used as the return value of getBackgroundColour in the Support lib. Anyway, the support lib also has plenty of other bugs (I'll make another pull request). Changing the header should cause the lib to recompile, and it is statically linked with plugins, so everything should be OK. This struct never goes through the OpenFX API.